test/build/50_dockerfile.bats

```diff
@@ -375,6 +373,61 @@ EOF
 }
 
 
+@test 'Dockerfile: SHELL' {
+   scope standard
+   [[ $CH_BUILDER = none ]] && skip 'no builder'
+   [[ $CH_BUILDER = buildah* ]] && skip "Buildah doesn't support SHELL"
+  
+   # tests that SHELL command can change executables and paramaters
+   run ch-build -t foo --no-cache -f - . <<'EOF'
+FROM 00_tiny
+RUN echo "1" $0
+SHELL ["/bin/ash","-c"]
+RUN  echo "2" $0
+SHELL ["/bin/sh", "-v", "-c"]
+RUN echo "3" $0
+EOF
+   echo "$output"
+   [[ $status -eq 0 ]]
+   [[ $output = *"1 /bin/sh"* ]]
+   [[ $output = *"2 /bin/ash"* ]]
+   [[ $output = *"3 /bin/sh"* ]]
+
+   # tests that it fails if shell doesn't exist
+   run ch-build -t foo -f - . <<'EOF'
+FROM 00_tiny
+SHELL ["/usr/bin/doesnotexist", "-c"]
+RUN print("hello")
+EOF
+   echo "$output"
+   [[ status -eq 1 ]]
```

Comment:
Were you able to figure out an error message that would tell us the failure was due to nonexistent shell?